src/test/scenarios/mixed-reality/configuration/readme.az.md

```diff
@@ -0,0 +1,36 @@
+## Azure CLI
+
+These settings apply only when `--az` is specified on the command line.
+
+``` yaml $(az)
+az:
+  extensions: mixed-reality
+  package-name: azure-mgmt-mixedreality
+  namespace: azure.mgmt.mixedreality
+az-output-folder: $(azure-cli-extension-folder)/src/mixed-reality
+python-sdk-output-folder: "$(az-output-folder)/azext_mixed_reality/vendored_sdks/mixedreality"
+
+directive:
+    - where:
+          group: mixed-reality spatial-anchor-account
+      set:
+          group: spatial-anchors-account
+    - where:
+          group: mixed-reality remote-rendering-account
+      set:
+          group: remote-rendering-account
+
+cli:
+    cli-directive:
+        - where:
+              group: "*"
+              op: "*"
+          hidden: true
+        - where:
+              group: RemoteRenderingAccounts
+              op: "*"
+          hidden: false
+        - where:
+              group: SpatialAnchorsAccounts
+              op: RegenerateKeys
+          hidden: false
```

Comment:
According to the config here, it's trying to incremental generate the `RemoteRenderingAccounts` operation group and `SpatialAnchorsAccounts RegenerateKeys` command.  But from the basecli folder the operation group `remote-rendering-account` and the command `spatial-anchors-account regenerate-key` are already exists https://github.com/Azure/autorest.az/pull/519/files?file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.py&file-filters%5B%5D=.ts#diff-23b07319e7c2a8ce96643a8cd6c05f08R16-R24
Our incremental codegen now is assumed for new operation group and new operation right?

Reply:
I changed, and please check again`